content/tools/general.md

```diff
@@ -25,6 +25,10 @@ It offers several [packages](https://atom.io/packages/search?q=cucumber) you can
 [TextMate](https://macromates.com/) is a text editor for macOS.
 See the [`Cucumber.tmbundle`](https://github.com/cucumber/cucumber-tmbundle) documentation.
 
+## Tidy Gherkin
```

Comment:
This tool should be on the [Related Tools](http://docs.cucumber.io/tools/related-tools/) page instead of the general tools page. Could you move it there please?

Reply:
Sure, I will move that.